src/core/utility/file/filePath/FilePath.h

```diff
@@ -1,22 +1,21 @@
 #pragma once
 /**
  * @file FilePath.h
- * @brief A robust file path management class using Boost filesystem
+ * @brief A robust file path management class using standard filesystem
  *
  * @details This class provides comprehensive file path manipulation and query capabilities,
- * offering a high-level abstraction over boost::filesystem::path with additional
+ * offering a high-level abstraction over fs::path with additional
  * functionality for path operations, validation, and environment variable expansion.
  *
  * @note The class is thread-safe for const methods but not for mutation operations
  */
 #include <filesystem>
 #include <memory>
+#include <optional>
 #include <string>
 #include <vector>
 
-namespace boost::filesystem {
-class path;
-}    // namespace boost::filesystem
+namespace fs = std::filesystem;
```

Comment:
Please don't expose fs namespace in the header.

Reply:
I agree.